docs/providers/kubeless/guide/deploying.md

```diff
@@ -95,7 +95,7 @@ provider:
       endpoint: http://minio.local
       bucket: workbench
       region: eu-central-1
-      s3ForcePathStyle: True
+      s3ForcePathStyle: true
```

Comment:
I believe this is a valid boolean in YAML, but when I was running `rg True` I found this and decided it should be changed as well. I'm not too attached if you want to remove it.

Reply:
Good call 👍